spec/models/round_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "rails_helper"
```

Comment:
There is some overall inconsistency with quotes. I'd suggest we set double quotes as the standard, if that's ok to you. 

We can apply rubocop's autofix for this in the future.

Reply:
Yes, right now is inconsistent with any option, let's discuss. But maybe we could merge and change everywhere in another PR?